README.md

```diff
@@ -1,2 +1,211 @@
-# stale-issues-finder
-Finds outdated issues and reports them
+# Stale Issue Finder 
+Finds outdated issues and generates an output data & message.
+
+Intended to be used with a notification action (Slack/Discord/Email/etc look at the example usage).
+
+Works great with the [`workflow_dispatch`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) or [`schedule`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) action events.
+
+## Why?
+
+This action is intended for the case where a repository (or an organization) needs to find out what issues have been stale for a while.
+
+By being agnostic on the result, users can use the output to generate a custom message on their favorite system.
+
+## Example usage
+
+You need to create a file in `.github/workflows` and add the following:
+
+```yml
+name: Find stale issues
+
+on:
+  workflow_dispatch:
+
+jobs:
+  fetch:
+    permissions:
+        issues: read
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch issues from here
+        # We add the id to access to this step outputs
+        id: stale
+        uses: paritytech/stale-issues-finder@main
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # optional, how many days since the last action for it to be stale
+          # defaults to 5
+          days-stale: 10
+        # example showing how to use the content
+      - name: Produce result
+        run: |
+          echo "There are $AMOUNT stale issues in this repository"
+          echo "$ACTION_ISSUES"
+        env:
+          # a number with the amount of stale issues in the repository
+          AMOUNT: ${{ steps.stale.outputs.stale }}"
+          # a formatted markdown message
+          ACTION_ISSUES: ${{ steps.stale.outputs.message }}"
+```
+
+### Inputs
+You can find all the inputs in [the action file](./action.yml) but let's walk through each one of them:
+
+- `GITHUB_TOKEN`: Token to access to the repository issues. If you are refering to a different repository this need to be a PERSONAL ACCESS TOKEN
+  - **required**
+  - If using on the same repo, you can simply use `${{ github.token }}`.
+- `repo`: name of the repository. Example: `https://github.com/paritytech/REPO-NAME-GOES-HERE`
+  - **defaults** to the repo where this action is ran.
+  - If set, you also need to set `owner`.
```

Comment:
Why not default the `owner` to the same owner of the repo where the action is run?
I imagine it would be mostly `paritytech` then?

Reply:
Good idea. Will implement that.